ls1intum/Artemis
# Added markdown for the lectures' description

### Checklist

### Motivation and Context



Fixes #836 

### Description

Added markdown for the lectures' description - #836. 

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Click on Lectures (on an existing course) or create a new course
4. If there is no lecture, click on Create Lecture. If there is, edit it. 
5. At this point, (*new*) you should be able to add formatting (and not simply text) to the Description.
6. You should be able to see (*new*) the formatted Description when
- (1) you click on Lectures (from the chosen course) &
- (2) you click on Attachments/ View (after clicking on Lectures)
7. The (*new*) the formatted Description should be also shown on the students Overview page:
- (3) Open Overview -> Click on Course -> Open Lectures (tab next to Exercises) -> Click on Lecture

### Screenshots



click on Lectures
![click on Lectures](https://user-images.githubusercontent.com/33368508/67623105-c4bf6e80-f821-11e9-9644-2feeb4ea4649.png)

click on View
![click on View](https://user-images.githubusercontent.com/33368508/67623108-c852f580-f821-11e9-86a3-e8ea9799c2c0.png)

click on Attachments
![click on Attachments](https://user-images.githubusercontent.com/33368508/67758455-fdd42a80-fa3d-11e9-8680-a84cba6494a2.png)

students Overview page
![students overview page](https://user-images.githubusercontent.com/33368508/68593250-46a3ce80-0495-11ea-8a81-25c1f9a791c2.png)
